org.telesculptor.TeleSculptor.yaml

```diff
@@ -0,0 +1,68 @@
+app-id: org.telesculptor.TeleSculptor
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: TeleSculptor
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
```

Comment:
It doesn't seem to use portals so it need at least `--filesystem=home`.

Reply:
I add it.

Reply:
and add lapack